modules/nf-core/universc/main.nf

```diff
@@ -0,0 +1,117 @@
+process UNIVERSC {
+    tag "$meta.id"
+    label 'process_medium'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the Cell Ranger tool. Please use docker or singularity containers."
+        conda (params.enable_conda ? "hcc::cellranger=3.0.2" : null)
+    }
+    container "tomkellygenetics/universc:1.2.4"
+    containerOptions = "--user root"
+
+    input:
+    tuple val(meta), path(reads)
+    path  reference
+
+
+    output:
+    tuple val(meta), path("sample-${meta.id}/outs/*"), emit: outs
+    path "versions.yml"                              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def sample_arg = meta.samples.unique().join(",")
+    def reference_name = reference.name
+    def input_reads = meta.single_end ? "--file $reads" : "-R1 ${reads[0]} -R2 ${reads[1]}"
+    """
+    bash /universc/launch_universc.sh \\
```

Comment:
Is the script not executable and in the path?
```suggestion
    launch_universc.sh \\
```

Reply:
Yes